### PR TITLE
[CLOUD-2737] JWS 5.0 OpenShift image

### PR DESCRIPTION
**Pull request by bekionel**
@rcernich Could you please take a look? Thanks!

### CODE REVIEW

File: templates/jws50-tomcat9-basic-s2i.json [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+            "description": "An example JBoss Web Server application. For more information about using this template, see https://github.com/jboss-openshift/application-templates.",
+            "template.openshift.io/long-description": "This template defines resources needed to develop Red Hat JBoss Web Server 5.0 Apache Tomcat 9 based application, including a build configuration, and an application deployment configuration.",
+            "template.openshift.io/documentation-url": "https://access.redhat.com/documentation/en/red-hat-jboss-web-server/",
+            "template.openshift.io/support-url": "https://access.redhat.com"
+        },
+        "name": "jws50-tomcat9-basic-s2i"
+    },
+    "labels": {
+        "template": "jws50-tomcat9-basic-s2i",
+        "jws": "1.0"
```
Comment (rcernich) [inline templates/jws50-tomcat9-basic-s2i.json]: maybe `jws50`, to distinguish from other versions?

Comment (bekionel) [reply templates/jws50-tomcat9-basic-s2i.json]: Changed the label to jws50.

File: templates/jws50-tomcat9-https-s2i.json [hunk 2]
```diff
@@ -0,0 +1,442 @@
...
+            "template.openshift.io/long-description": "This template defines resources needed to develop Red Hat JBoss Web Server 5.0 Apache Tomcat 9 based application, including a build configuration, application deployment configuration, and secure communication using https.",
+            "template.openshift.io/documentation-url": "https://access.redhat.com/documentation/en/red-hat-jboss-web-server/",
+            "template.openshift.io/support-url": "https://access.redhat.com"
+
+        },
+        "name": "jws50-tomcat9-https-s2i"
+    },
+    "labels": {
+        "template": "jws50-tomcat9-https-s2i",
+        "jws": "1.0"
```
Comment (rcernich) [inline templates/jws50-tomcat9-https-s2i.json]: We should make these all consistent, so jws or jws50.  Sorry I wasn't clear in my previous comment.  Same should apply to the image streams.

Comment (bekionel) [reply templates/jws50-tomcat9-https-s2i.json]: @rcernich Made the appropriate changes, I hope I understood your comment correctly.

**Review by rcernich — APPROVED** [review]
LGTM

**Pull request merged**